src/undux/GameStore.js

```diff
@@ -21,6 +21,7 @@ type State = {|
   nearbyVenues: Set<string>, // venue ids
 
   jpDisplayStyle: JpDisplayStyleType,
+  playerID: string,
```

Comment:
this should be `?string` so we can make sure to handle things properly when the player id is null